### ISSUE TITLE
Preview pane stopped updating

### ISSUE THREAD

**Issue by svartrev** [fabiocolacio/Marker#169]
While I was typing some text, the preview pane suddenly stopped updating. I have no idea what triggered this, and I found no way to re-enable it. I would provide more information if I knew what to look for. I haven't had it happen again, and have no idea how to re-create this.

Marker version 2018-03-09 running on Arch Linux.

**Comment by fabiocolacio** [comment]
Hi @svartrev.

Thanks for the input; bug reports are highly appreciated!

How did you install Marker? Did you compile it from source, use the AUR, or use the flatpak?

Also, when you ran into this problem, did the name of the document you were editing contain any characters with accents, or emojis? If it did, this could be related to a known bug, #136.

**Comment by svartrev** [comment]
Hi @fabiocolacio 

Thanks for the prompt response.

I installed Marker from the AUR.
I was editing a new document, straight after opening Marker for the first time. As I hadn't saved any documents yet, there was no file name to have any special characters in.
Also, I don't think I had typed any special characters into the editor, it was all normal text and markdown, and some html tags, I think.

It has happened once since, although I can't trigger it again. There was nothing but normal text in the editor the second time.

**Comment by fabiocolacio** [comment]
As I have not been able to reproduce this, there is not much that I can do about this for now.

If you encounter it again, the following information can be useful to solving it:

1. Try the ``Ctrl``+``r`` shortcut, which manually refreshes, and let me know if that bypasses the issue.
2. Send us the contents of the document. This helps us know if it is related to the contents of the document or not. Although it is doubtful, since you mentioned it was plain text, it still has a chance of leading us somewhere. Of course, only share this with us if the document does not contain sensitive or personal information. Your privacy is more important than this bug :)

@Mandarancio have you ever encountered this issue?

**Comment by Mandarancio** [comment]
I tried as well to reproduce the bug on my arch setup but I did not experienced the issue... I will try and look on what could trigger such behaviour.

**Comment by Mandarancio** [comment]
@svartrev have you experimented the problem again? Or it has disappear?

**Comment by svartrev** [comment]
Sorry for the slow response. Busy weekend :)

I tried again to see if I could reproduce it. I wasn't having any success, and was about to give up, but cleared the page (i.e. Ctrl + End, Ctrl + Shift + Home, Del), and started typing some random stuff, and it happened again. Pressing Ctrl + r didn't do anything. I pressed it several times, and even tried selecting the preview pane before pressing it.

I saved a screenshot, where you can see that the last few lines are not displayed in the preview pane, and the text is all normal stuff. I've also saved the text as a plain text file.

I haven't closed Marker yet, and may be able to keep it open for a while (shouldn't need to close it in the next day or two if nothing crashes), in case you want me to try anything else. After taking the screenshot, somewhere in all the minimizing and covering it with other windows, the preview pane has gone blank. I tried all the different view options (dual pane, dual window, preview only, editor only), and it comes back blank each time.

![screenshot_2018-04-09_14-25-23](https://user-images.githubusercontent.com/8195402/38497796-88821f4e-3c02-11e8-9084-0b3ca40a6af6.png)
[Marker text 2018-04-09-14-25-23.txt](https://github.com/fabiocolacio/Marker/files/1889873/Marker.text.2018-04-09-14-25-23.txt)

**Comment by megamorphf** [comment]
For me preview does not work as well.  Command I used:

    flatpak run com.github.fabiocolacio.marker -p './no-render.md'

 I also tried absolute path to a file, but still no rendering.  Everything is OK standard editor/preview mode, but not with the `-p` parameter. One thing I noticed is that the app is showing wrong notebook directory (see screenshot).

Installed using flatpak, on KaliLinux

![image](https://user-images.githubusercontent.com/13219260/39074574-b4eed724-44fa-11e8-9561-911c34f92c57.png)
